# Introduction
단기과열종목 지정예고 조건이 충족된 종목의 장중매매 수익률을 어떨까?  
유튜브1: https://youtu.be/8L-ApqFCG3Y  
유튜브2: https://youtu.be/g_YtTMQDJzM

# Import

In [23]:
import pandas as pd
import numpy as np
from matplotlib import pyplot as plt
%matplotlib inline

# Load

In [24]:
df = pd.read_csv('data.csv')
df.head()

주문일자     종목명  매수가격      매수시간      매도시간   수익률    종목코드  뉴스기사 최대거래대금 시간  \
0  2022-03-29   고려시멘트  5230  09:09:07  09:21:27  2.87  198440     1  09:06:00   
1  2022-03-29   고려시멘트  5410  09:28:09  09:32:19  1.85  198440     1  09:06:00   
2  2022-03-29   고려시멘트  5440  10:38:18  10:51:17  0.15  198440     1  09:06:00   
3  2022-03-29    영진약품  5490  09:51:11  09:55:24  0.66    3520     1  09:50:00   
4  2022-03-29  버킷스튜디오  5060  09:58:16  09:59:56 -0.59   66410     1  09:22:00   

   최대거래대금  매수등락률   단기과열  
0     113  16.22   True  
1     113  20.22   True  
2     113  20.89   True  
3      62   6.60  False  
4      72  11.58  False

# Preprocessing

In [26]:
df = df[df['뉴스기사'] == 1]
df = df[df['매수등락률'] > 12]
df = df[df['최대거래대금'] > 60]

# Mean

In [19]:
df[df['단기과열'] == True]['수익률'].mean()

1.6091666666666666

In [20]:
df[df['단기과열'] == False]['수익률'].mean()

0.8299999999999998

# Median

In [21]:
df[df['단기과열'] == True]['수익률'].median()

2.09

In [22]:
df[df['단기과열'] == False]['수익률'].median()

0.265

# Results
단기과열 조건에 해당하는 종목의 수익률은 mean 1.6%, median 2.09%이고  
단기과열 조건에 해당하지 않는 종목의 수익률은 mean 0.82%, median 0.26%이다.

# Conclusion
상식에 반하는 결과로 단기과열 조건을 충족시킨 종목에서 수익이 더 잘난다.   
아마도 광기로 급등하는 종목은 단타거래 참여자도 많고 변동폭도 커서 수익이 잘 나는 것 같다.  
따라서 단기과열조건은 신경 쓰지 않고 매매하면 될 것 같다.  
주의할 점은 갭하락 위험이 있는 종가베팅이다.  
오버나잇만 하지 않는다면 오히려 수익이 잘 난다.